libs/block-editor/src/lib/extensions/components/bubble-menu-link-form/bubble-menu-link-form.component.html

```diff
@@ -0,0 +1,50 @@
+
+<div
+    class="overlay"
+    (click)="hideForm()"
+    #formContainer
+>
+    <form
+        autoComplete="off"
+        (click)="preventHideOnFormClick( $event )"
+        (mousedown)="focusInput()"
+        (submit)="addLink()"
+        #form
+    >
```

Comment:
You don't need to nest the form inside the overlay that is what causes that you have to do `preventHideOnFormClick`.

But now I think, shouldn't we listen to a click or a focus in the editor to hide our things? instead of creating a div just for this?

Reply:
The purpose of a div is to avoid clicking something else when the `add-link-form` is ope

https://user-images.githubusercontent.com/72418962/142477447-985a0eae-27d9-484d-9e49-a04c3922faaa.mov

n.

Reply:
Exactly, the `div` is to have an area to click and close the form but because you put the form INSIDE this div then clicks in the form will trigger clicks in the div and by that means the form close and in order to prevent that you had to add a click to the form to stoppropagation, instead of this you can just move the form outside the div and you will not have to care for that propagation.

But also, do we need the div or can we just listen for click in the editor itself?